content/en/examples/admin/konnectivity/egress-selector-configuration.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiserver.k8s.io/v1beta1
 kind: EgressSelectorConfiguration
 egressSelections:
 # Since we want to control the egress traffic to the cluster, we use the
-# "cluster" as the name. Other supported values are "etcd", and "master".
+# "cluster" as the name. Other supported values are "etcd", and "controlplane".
```

Comment:
The ref commit is here https://github.com/kubernetes/kubernetes/commit/a0aebf96ec2eef6517e2611335f0e6c9375dd807

```go
# file: staging/src/k8s.io/apiserver/pkg/server/egressselector/egress_selector.go
func lookupServiceName(name string) (EgressType, error) {
	switch strings.ToLower(name) {
	// 'master' is deprecated, interpret "master" as controlplane internally until removed in v1.22.
	case "master":
		klog.Warning("EgressSelection name 'master' is deprecated, use 'controlplane' instead")
		return ControlPlane, nil
	case "controlplane":
		return ControlPlane, nil
	case "etcd":
		return Etcd, nil
	case "cluster":
		return Cluster, nil
	}
	return -1, fmt.Errorf("unrecognized service name %s", name)
}
```


BTW, the "master" value is also accepted but deprecated, it will automatically convert to "controlplane". 

Please take a look at `staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/conversion.go`
```go
func Convert_v1alpha1_EgressSelection_To_apiserver_EgressSelection(in *EgressSelection, out *apiserver.EgressSelection, s conversion.Scope) error {
	if err := autoConvert_v1alpha1_EgressSelection_To_apiserver_EgressSelection(in, out, s); err != nil {
		return err
	}
	if out.Name == "master" {
		out.Name = "controlplane"
	}
	return nil
}
```

Reply:
Thanks for the information!